docs/docs/deploy/environment-variables.mdx

```diff
@@ -0,0 +1,131 @@
+| `TF_DETERMINISTIC_OPS`            | Configures TensorFlow ops to run deterministically. See [Configuring TensorFlow](../nlu-based-assistants/tuning-your-model.mdx#configuring-tensorflow).                                      | `false`   |
+
+### Observability
+
+| Environment Variable         | Description                                                                                                       | Default   |
+| :--------------------------- | :---------------------------------------------------------------------------------------------------------------- | :-------- |
+| `LOG_LEVEL`                  | The log level for Rasa and Rasa Pro.                                                                              | `"INFO"`  |
+| `LOG_LEVEL_LIBRARIES`        | The log level for 3rd-party libraries. More info [here](../command-line-interface.mdx#log-level).                 | `"ERROR"` |
+| `LOG_LEVEL_KAFKA`            | The log level for `kafka` library. More info [here](../command-line-interface.mdx#log-level).                     | `"ERROR"` |
+| `LOG_LEVEL_RABBITMQ`         | The log level for `rabbitmq` library. More info [here](../command-line-interface.mdx#log-level).                  | `"ERROR"` |
+| `LOG_LEVEL_FAKER`            | The log level for `faker` library. More info [here](../command-line-interface.mdx#log-level).                     | `"ERROR"` |
+| `LOG_LEVEL_PRESIDIO`         | <RasaProLabel /> The log level for `presidio` library. More info [here](../command-line-interface.mdx#log-level). | `"ERROR"` |
+| `RASA_TELEMETRY_ENABLED`     | Toggle telemetry reporting. More info [here](../telemetry/telemetry.mdx)                                          | `true`    |
+| `RASA_TELEMETRY_DEBUG`       | Toggle debug information for telemetry reporting.                                                                 | `false`   |
+| `RASA_PRO_TELEMETRY_ENABLED` | <RasaProLabel /> Toggle telemetry reporting. More info [here](../telemetry/telemetry.mdx)                         | `true`    |
+| `RASA_PRO_TELEMETRY_DEBUG`   | <RasaProLabel /> Toggle debug information for telemetry reporting.                                                | `false`   |
+| `TRACING_SERVICE_NAME`       | <RasaProLabel /> The top-level service name when sending traces. More info [here](../operating/tracing.mdx)       | `"rasa"`  |
+
+### Studio
+
+| Environment Variable             | Description                 | Default   |
+| :------------------------------- | :-------------------------- | :-------- |
+| `RASA_STUDIO_AUTH_SERVER_URL`    | <RasaProLabel /> TBD        |           |
+| `RASA_STUDIO_CLI_STUDIO_URL`     | <RasaProLabel /> TBD        |           |
+| `RASA_STUDIO_CLI_REALM_NAME_KEY` | <RasaProLabel /> TBD        |           |
+| `RASA_STUDIO_CLI_CLIENT_ID_KEY`  | <RasaProLabel /> TBD        |           |
```

Comment:
I think we can remove this section or move these envars to a section that's private envars meant for internal use

Reply:
but are there really private? Studio is going to be deployed by users at a specific URL, so they will probably need to configure these variables. Or am I mistaken?

Reply:
I don't have enough context to comment but last time I spoke to Radovan, he said that said these shouldn't be externally configurable. [Here's the slack discussion](https://rasa-hq.slack.com/archives/C01HHMR4X8S/p1690383497623729?thread_ts=1689865560.709779&cid=C01HHMR4X8S)

Reply:
I read the thread and yes, they don't need it in the helm charts. But they're required for `rasa studio` commands to work locally. We will need to document these environment variables I think. I'll create a task to do this before the release and remove this section, just in case it's not done before the release.

Reply:
https://rasahq.atlassian.net/browse/ATO-1653

Reply:
a39be9b5fdb